src/tool_fsharp/tool_fsc.csproj

```diff
@@ -0,0 +1,68 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.tasks" />
+
+  <PropertyGroup>
+    <VersionPrefix>$(CliVersionPrefix)</VersionPrefix>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeFrameworkVersion>$(CLI_SharedFrameworkVersion)</RuntimeFrameworkVersion>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <PublishDir>$(FSharpDirectory)</PublishDir>
+    <VersionSuffix>$(CommitCount)</VersionSuffix>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NetCore.App"                Version="$(CLI_SharedFrameworkVersion)" />
+    <PackageReference Include="microsoft.fsharp.core.netcore"        Version="$(CLI_FSharp_Version)" />
```

Comment:
I see that https://github.com/Microsoft/visualfsharp contains a `Microsoft.FSharp.Core.netcore.nuspec` file and a Microsoft.FSharp.Compiler.netcore.nuspec` file. This repo seems to require mono to build.